cookbooks/nagios/files/default/plugins/check_batch.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+
+import argparse
+import logging
+from time import time
+from sys import exit
+
+logging.basicConfig(level=logging.INFO,format='%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+
+try:
+    import boto3
+except ImportError as e:
+    boto3 = None
+
+STATUS = {
+    "OK": 0,
+    "WARNING": 1,
+    "ERROR": 2,
+    "UNKNOWN": 3
+}
+
+def nagios_exit(status_name, message):
+    print("{}: {}".format(status_name, message))
+    exit(STATUS[status_name])
+
+def createdAt(elem):
+    return elem["createdAt"]
+
+def checkFailed(batch_client, queue_name, warn_limit, error_limit):
+
+    # Failed jobs
+    failed = batch_client.list_jobs(jobQueue=queue_name, jobStatus="FAILED")["jobSummaryList"]
+    if len(failed) >= error_limit:
+        nagios_exit("ERROR", "Too many jobs are failing in queue: %s" % queue_name)
+    elif len(failed) >= warn_limit:
+        nagios_exit("WARNING", "%i jobs failed in queue: %s" % [len(failed), queue_name])
+
+def checkBlocked(batch_client, queue_name, warn_limit, error_limit):
+
+    # Blocked queue
+    runnable = batch_client.list_jobs(jobQueue=queue_name, jobStatus="RUNNABLE")["jobSummaryList"]
+    if len(runnable) > 0:
+        starting = batch_client.list_jobs(jobQueue=queue_name, jobStatus="STARTING")["jobSummaryList"]
+        running = batch_client.list_jobs(jobQueue=queue_name, jobStatus="RUNNING")["jobSummaryList"]
+        active_count = len(runnable) + len(starting) + len(running)
+        runnable.sort(key=createdAt)
+        runnable_age_of_oldest = (int(round(time() * 1000)) - runnable[0]["createdAt"]) // (1000 * 60 * 60)  # Hours
+        if runnable_age_of_oldest >= error_limit:
+            nagios_exit("ERROR", "Job(s) are blocked in queue: %s" % queue_name)
+        elif runnable_age_of_oldest >= warn_limit and active_count > 1:
```

Comment:
should use same logic as previous the if statment but with a different limit

```suggestion
        elif runnable_age_of_oldest >= warn_limit:
```